SDWebImage/UIView+WebCache.m

```diff
@@ -189,7 +201,7 @@ - (void)sd_internalSetImageWithURL:(nullable NSURL *)url
 }
 
 - (void)sd_cancelCurrentImageLoad {
-    [self sd_cancelImageLoadOperationWithKey:NSStringFromClass([self class])];
```

Comment:
This behavior change seems have no document update, and it's not suitable in a patch version release.

Reply:
Emm, it's a bug fix typically, because we can't hard code the operation key as the current operation key. Think about `UIButton`, it not use ` [self class]` to load image at all.